spec/src/main/asciidoc/changelog.asciidoc

```diff
@@ -21,12 +21,17 @@
 
 === Release 2.0
 
+==== Incompatible changes
+
+* Update OpenTracing API to 0.33.0 (https://github.com/eclipse/microprofile-opentracing/pull/177[#177])
```

Comment:
Can you highlight what api changes are incompatible changes such as method removal and offer some solutions how to help with the migration, such as use an alternative methods?